src/App.svelte

```diff
@@ -36,9 +36,17 @@
 	})
 </script>
 
+<style>
+	.kui-app {
+		height: 100vh;
+		display: flex;
```

Comment:
Well done! Even though it is not necessary, this makes it future proof, in case KaiOS creates some devices with different resolution/screen size. :)

Reply:
Yes! Even though flex does not work 100% in current kaios version now (for instance, "Settings" is still hidden behind bottom menu in kaios, and not in web version) I think this is the right approach going forward.